<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Тестовая задача ЕГК   v1.5.0 alpha1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="solutionPainterFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>518</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать вычисления</string>
    </property>
   </widget>
   <widget class="QToolBox" name="SettingsToolBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>221</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="problemSettingsPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>420</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Параметры задачи</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::WrapAllRows</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>Отношение высоты области к ее ширине:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="heightComboBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>0,25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0,5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="prandtlLabel">
         <property name="text">
          <string>Число Прандтля:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="prandtlDoubleSpinBox">
         <property name="toolTip">
          <string>Введите число от 0,001 до 1000</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.001000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="grashofLabel">
         <property name="text">
          <string>Число Грасгофа:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="grashofDoubleSpinBox">
         <property name="toolTip">
          <string>Введите число от 0,001 до 100000</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="minimum">
          <double>0.001000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="tBCLabel">
         <property name="text">
          <string>Краевые условия для температуры:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="tBCComboBox">
         <item>
          <property name="text">
           <string>Подогрев сбоку</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Подогрев снизу</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="settingsWallsGroupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>221</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Жесткие стенки</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="leftWallCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Левая</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="rightWallCheckBox">
          <property name="text">
           <string>Правая</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="topWallCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Верхняя</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="bottomWallCheckBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Нижняя</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Параметры решателя</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="rowWrapPolicy">
        <enum>QFormLayout::WrapAllRows</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nxLabel">
         <property name="text">
          <string>Число узлов сетки по горизонтали:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="nxSpinBox">
         <property name="toolTip">
          <string>Введите число от 20 до 400</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
         </property>
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nyLabel">
         <property name="text">
          <string>Число узлов сетки по вертикали:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="nySpinBox">
         <property name="toolTip">
          <string>Введите число от 20 до 400</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
         </property>
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>400</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="maxIterLabel">
         <property name="text">
          <string>Максимальное число итераций:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="maxIterSpinBox">
         <property name="toolTip">
          <string>Введите число от 100 до 300000</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>300000</number>
         </property>
         <property name="value">
          <number>5000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="wTLabel">
         <property name="text">
          <string>Коэффициент релаксации для T</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="wTDoubleSpinBox">
         <property name="toolTip">
          <string>Введите число от 0 до 2</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="wPsiLabel">
         <property name="text">
          <string>Коэффициент релаксации для ψ</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="wPsiDoubleSpinBox">
         <property name="toolTip">
          <string>Введите число от 0 до 2</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="wOmegaLabel">
         <property name="text">
          <string>Коэффициент релаксации для ω</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="wOmegaDoubleSpinBox">
         <property name="toolTip">
          <string>Введите число от 0 до 2</string>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.300000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Управление отрисовкой результата</string>
     </attribute>
     <widget class="QGroupBox" name="settingsPaintGroupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Цель для отрисовки</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QRadioButton" name="paintTButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>T</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="paintPsiButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ψ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="paintOmegaButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ω</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="paintVxButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Vx</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="paintVyButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Vy</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="saveImageButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить изображение в файл</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="solutionProgressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>581</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="solutionProgressLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Прогресс решения: Итерация - из - ; Текущая невязка - </string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>542</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Прервать вычисления</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="separator"/>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Настройки</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="actionPrGr"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <action name="aboutQtAction">
   <property name="text">
    <string>О Qt...</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>Справка...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionPrGr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ограничить значения Pr и Gr</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
